src/gatsby-ssr.spec.ts

```diff
@@ -30,10 +32,16 @@ describe('gatsby-ssr.js', () => {
         return []
       }
 
-      onPreRenderHTML({ getHeadComponents, replaceHeadComponents: replaceHeadComponentsSpy, getPostBodyComponents, replacePostBodyComponents: replacePostBodyComponentsSpy }, {})
+      onPreRenderHTML({ getHeadComponents, pathname, replaceHeadComponents: replaceHeadComponentsSpy, getPostBodyComponents, replacePostBodyComponents: replacePostBodyComponentsSpy }, {})
       expect(replaceHeadComponentsSpy).toHaveBeenCalledWith([headComponentsData[2], headComponentsData[3]])
     })
 
+    it('does the pathname match any of the exclusions', () => {
```

Comment:
This is great but could you move this test to a new `describe('checkPathExclusion')` since it is not actually testing `onPreRenderHTML` here.

Or change the test to test how checkPathExclusion works in the flow of `onPreRenderHTML`

Or both ;)